node-bb/README.md

```diff
@@ -0,0 +1,177 @@
+This document is instructions on how to build and run the Docker container of the forums using an 
+[official Docker container of NodeBB](https://hub.docker.com/r/nodebb/docker) as the base image.
+
+## Building
+
+1. Make a directory on your machine for where the config file and upload folder will live. ```mkdir -p /opt/triple-forums```
+
+2. Make a directory for uploaded files. Inside the upload directory, create folders category, emoji, files, profile, 
+sounds, and system.
+
+        mkdir -p /opt/triple-forums/uploads/category
+        mkdir -p /opt/triple-forums/uploads/emoji
+        mkdir -p /opt/triple-forums/uploads/files
+        mkdir -p /opt/triple-forums/uploads/profile
+        mkdir -p /opt/triple-forums/uploads/sounds
+        mkdir -p /opt/triple-forums/uploads/system
+
+3. Build the Docker image using the command ```docker build -t triplea-forums:latest .```
+
+### Running
+
+#### First time run
+
+You will need a running Mongo database locally or in a Docker container. See [MongoDB](#mongodb) for how to do this.
+
+1. Run the triplea-forums:latest image using the command 
+```docker run -d --rm --name triplea-forums -v /opt/triple-forums/uploads:/usr/src/app/public/uploads -p 8080:4567 triplea-forums:latest```
+
+2. Navigate to [http://0.0.0.0:8080/](http://0.0.0.0:8080/). This is the install and configuration screen for NodeBB.
+
+3. Fill out the details on the page.
+
+4. Once the configuration is complete, you will need to save the config.json file off the container to the folder you 
+created earlier. Run ```docker container cp triplea-forums:/usr/src/app/config.json /opt/triplea-forums```
+
+#### Second and later runs
+
+1. If you did not save the config.json from before, you can make a copy of the example/config.json 
+and update it per the instructions in [Configuration](#configuration) section. 
+```cp example/config.json /opt/triplea-forums/config.json```
+
+2. With the config.json run the triplea-forums:latest image using the command 
+```docker run -d --rm --name triplea-forums -v /opt/triple-forums/config.json:/usr/src/app/config.json -v /opt/triple-forums/uploads:/usr/src/app/public/uploads -p 8080:4567 triplea-forums:latest```
+
+### Stopping
+
+To stop the running container, run the following: ```docker stop triplea-forums```
+
+### MongoDB  
+
+Mongo is the database that backs triplea-forums. You can install and run a local copy by following the
+instructions [here](https://docs.mongodb.com/manual/installation/) or you can run a Docker image with Mongo already
+installed.
+
+Before running the Docker image it is important to know which version of Mongo you want to run. As of this writing the 
+latest version is 4.4. In most cases running that will be fine however if you need a different version, you can find 
+the list of Docker image versions on [https://hub.docker.com/_/mongo](https://hub.docker.com/_/mongo).
+
+#### Mac OSX and Windows
+
+This command will start the Docker container:  
+```shell script
```

Comment:
Codacy found an issue: [[code-block-style] Code blocks should be indented](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6342386)